### PR TITLE
[1.7.10] Add Item-Group-Api

### PR DESCRIPTION
**Pull request by Domaman202**
I do not know why, but my module does not load and does not appear in the list of mods, who knows why?..(

### CODE REVIEW

File: modules/item-groups/src/main/java/net/legacyfabric/fabric/api/client/itemgroup/FabricItemGroupBuilder.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package net.legacyfabric.fabric.api.client.itemgroup;
+
+import net.legacyfabric.fabric.impl.item.group.ItemGroupExtensions;
+import net.minecraft.item.Item;
+import net.minecraft.item.itemgroup.ItemGroup;
+import net.minecraft.util.Identifier;
+
+public class FabricItemGroupBuilder {
+	private Identifier identifier;
```
Comment (arthurbambou) [inline modules/item-groups/src/main/java/net/legacyfabric/fabric/api/client/itemgroup/FabricItemGroupBuilder.java:9]: why are the fields here private?

**Comment by arthurbambou** [comment]
also you need to launch the task called "licenseFormatAll"

File: modules/item-groups/src/main/java/net/legacyfabric/fabric/api/client/itemgroup/FabricItemGroupBuilder.java (hunk 1 above)
Comment (Domaman202) [reply modules/item-groups/src/main/java/net/legacyfabric/fabric/api/client/itemgroup/FabricItemGroupBuilder.java:9]: in the original fabric api it was like this =)

**Comment by Domaman202** [comment]
ok thx

**Comment by Domaman202** [comment]
And 
"I do not know why, but my module does not load and does not appear in the list of mods, who knows why?..("
Screenshot_22

**Comment by Domaman202** [comment]
i reload project and all ok, thx

File: modules/item-groups/src/main/java/net/legacyfabric/fabric/api/client/itemgroup/FabricItemGroupBuilder.java (hunk 1 above)
Comment (arthurbambou) [reply modules/item-groups/src/main/java/net/legacyfabric/fabric/api/client/itemgroup/FabricItemGroupBuilder.java:27]: you should look at this version of item group api I think.
https://github.com/FabricMC/fabric/tree/1.16/fabric-item-groups-v0

**Review by arthurbambou — CHANGES_REQUESTED** [review]

**Review by arthurbambou — APPROVED** [review]

**Review by BoogieMonster1O1 — CHANGES_REQUESTED** [review]
Please add a test mod

**Comment by Domaman202** [comment]
And how to run test-mod?...

**Review by arthurbambou — APPROVED** [review]

**Review by BoogieMonster1O1 — APPROVED** [review]
Almost ready for merge. Can you send a screenshot of the testmod working when you run minecraft?

**Comment by BoogieMonster1O1** [comment]
> And how to run test-mod?...

If you're using IntelliJ idea, edit the Minecraft Client run configuration and make it use the classpath of the testmod sourceset rather than the main sourceset

**Comment by Domaman202** [comment]
https://user-images.githubusercontent.com/51743763/169692877-910299c1-3692-4edb-b489-f82674b538a3.mp4

**Comment by Domaman202** [comment]
Screenshot_29

**Pull request merged**